### ISSUE TITLE
Samples crash with exception

### ISSUE THREAD

**Issue by urcereal** [elnormous/ouzel#47]
Visual Studio 2017, Windows 10 SDK 10.0.18362.0
        msvcp140.dll!00007ff9c96b2e70()	Unknown
>	[Inline Frame] samples.exe!std::_Mtx_lockX(_Mtx_internal_imp_t *) Line 72	C++
 	[Inline Frame] samples.exe!std::_Mutex_base::lock() Line 49	C++
 	[Inline Frame] samples.exe!std::unique_lock<std::mutex>::{ctor}(std::mutex &) Line 150	C++
 	samples.exe!ouzel::graphics::Graphics::handleEvent(const ouzel::graphics::RenderDevice::Event & event) Line 161	C++
 	[Inline Frame] samples.exe!std::_Func_class<void,ouzel::graphics::RenderDevice::Event const &>::operator()(const ouzel::graphics::RenderDevice::Event &) Line 43	C++
 	samples.exe!ouzel::graphics::RenderDevice::process() Line 45	C++
 	samples.exe!ouzel::graphics::d3d11::RenderDevice::process() Line 322	C++
 	samples.exe!ouzel::graphics::d3d11::RenderDevice::renderMain() Line 1114	C++
 	[External Code]

**Comment by elnormous** [comment]
How can I reproduce the error? What did you do when you got it?

**Comment by urcereal** [comment]
I've cloned from your repository into a working directory. Retargeted the solution to my latest Windows 10 SDK and built the solution (x64, debug). This was a fresh clone so I wasn't updating an existing local copy. I also did a clean of the solution. I've tried this on 3 different Windows PCs all with the same result so I don't think it's with my environment . . . exception in xthread _Mtx_lockX. If I can supply any other info to help just let me know.
